DapperDoodle/BuilderHelpers.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.ComponentModel;
+using System.Data;
+using System.Text;
+using PluralizeService.Core;
+
+namespace DapperDoodle
+{
+    public static class BuilderHelpers
+    {
+        public static DataTable ObjectToDataTable(this Type obj)
+        {
+            var dt = new DataTable();
```

Comment:
minor: can use LINQ here (& suggest rename):

```csharp
public static DataTable DataTableForType(this Type type)
{
	return (type?.GetProperties() ?? new PropertyInfo[])
		.Aggregate(
			new DataTable(),
			(acc, cur) =>
			{
				acc.Columns.Add(cur.Name, cur.PropertyType);
				return acc;
			});
}
```

(assuming I didn't typo)

Reply:
Thank you. This is very helpful. 🙂